### PR TITLE
[JA] Add Ranking_Criteria/osu!mania

### PR DESCRIPTION
**Pull request by ViaSnake**
Add `Ranking_Criteria/osu!mania` in japanese.
This translation is based on the thread [日本語版 osu!mania ranking criteria](https://osu.ppy.sh/community/forums/topics/1285567) posted by RuberusuScarlet in the osu!forum
I have exchanged messages with RuberusuScarlet and received his permission to add this translation.

### CODE REVIEW

**Review by peppy — CHANGES_REQUESTED** [review]

File: wiki/Ranking_Criteria/osu!mania/ja.md [hunk 1]
```diff
@@ -1,69 +1,196 @@
...
+  - 完全に正確にプレイするのは無理でしょう。
+  - 一貫性のある予測可能なスナップで効果的に表現されます。
+  - 不適切な難易度の偏りを引き起こす可能性があります。
+- **ストーリーボードでサンプリングしたヒットサウンドを音符のヒットサウンド同じ音量で使うことは避けてください。** ストーリーボードのヒットサウンドは、ビートマップ内のオブジェクトからの音声フィードバックが埋もれたり歪める可能性があるため、ノーツにヒットサウンドを配置することを優先する必要があります。ただし、ストーリーボードのサンプルは、キーサウンドが曲から直接抽出される場合や、難易度が低く、必要なヒットサウンドをサポートするにはノーツが少なすぎる場合に使用できます。
+- **ヒットノーマルのサポートがない場合、キーサウンドを使用しないでください。** ヒットサウンドが曲と完全にマッチしている場合、フィードバックは最小限に抑えられます。ただし、曲から直接抽出されたキーサウンドを曲のオーディオの代わりに使用する場合は、ヒットノーマルサポートは必要ありません。
+- **コード内の複数の音に同じヒットサウンドを使わないでください。** これにより、必要以上のフィードバックを引き起こします。このガイドラインは、同じ場所で使用されているヒットサウンドとストーリーボードのサンプルヒットサウンドにも適用されます。
+- **タイミングの変更は、スライダー速度を調整することにより、統一されたスクロール速度を持つ必要があります。** これは、全てのタイミングセクションのスクロール速度をビートマップで最も使用されているBPMを基準に速度を合わせることができます。基準BPMは、曲の選択でBPM範囲の後に括弧内に表示されます。例: `BPM: 60-160 (130)`。このガイドラインは、SVエフェクトの作成に使用されるスライダー速度の変更を追加する機能には影響しないことに注意してください。
+- **スライダー速度の変化は、曲の一般的な強度に従う必要があります。**
+- **スライダーの速度の変更は、曲に使用されてるビートに応じて合わせる必要があります。** ストレートビートを使用している場合は、1/4、1/8、1/16を使用する必要があります。スウィングビートを使用している場合は、1/6または1/12を使用する必要があります。曲のビートが推奨位置と異なる場合は、該当するビートを使用が優先されます。
```
Comment (peppy) [inline wiki/Ranking_Criteria/osu!mania/ja.md:75]:
```suggestion
- **スライダーの速度の変更は、曲に使用されてるビートに応じて合わせる必要があります。** ストレートビートを使用している場合は、1/4、1/8、1/16を使用する必要があります。スウィングビートを使用している場合は、1/6または1/12を使用する必要があります。曲のビートが推奨位置と異なる場合は、該当するビートが優先して使用されます。
```

**Review by peppy — CHANGES_REQUESTED** [review]
re-review for updated terminology

**Comment by ViaSnake** [comment]
Please wait a bit before merging. I may still need to clean up this article.
I will self-review words and grammar, and replace it with more wiki-friendly words.

**Comment by ViaSnake** [comment]
Done.

**Review by peppy — CHANGES_REQUESTED** [review]
Just a minor one

**Pull request merged**